sormas-app/app/src/main/java/de/symeda/sormas/app/BaseEditActivity.java

```diff
@@ -178,6 +178,13 @@ protected void setStoredRootEntity(ActivityRootEntity entity) {
 		this.storedRootEntity = entity;
 	}
 
+	/**
+	 *
```

Comment:
Please add a comment to the comment (describing what the method is supposed to be used for) :)